packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -59,6 +60,62 @@ export function getBlogTags(blogPosts: BlogPost[]): BlogTags {
   }));
 }
 
+export function getBlogTagsPostPaginated(
+  blogPosts: BlogPost[],
+  postsPerPageOption: number | 'ALL',
+  baseUrl: string,
+  routeBasePath: string,
+  blogDescription: string,
+  blogTitle: string,
+): BlogTagPostPaginated[] {
+  const groups = groupTaggedItems(
+    blogPosts,
+    (blogPost) => blogPost.metadata.tags,
+  );
+
+  return flatMap(groups, (group) => {
```

Comment:
`flatMap` is banned for a reason 😅 Why not `groups.flatMap`?

Reply:
thanks for the correction, any context why the `flatMap` being banned?

Reply:
Because `Array#flatMap` works similar in every way to `flatMap(arr)`. If you need to flat an object, just use `Object.values(group).flatMap`. It's not worth importing lodash because no readability is sacrificed here.